tests/models/upernet/test_modeling_upernet.py

```diff
@@ -199,6 +199,11 @@ def test_save_load_fast_init_from_base(self):
     def test_save_load_fast_init_to_base(self):
         pass
 
+    @require_torch_multi_gpu
```

Comment:
Why add this annotator if we skip the test?

Reply:
It's fine without it. But we just keep the original context to make it clear - as we do in many other places, like

```
    @unittest.skip(reason="`saved_model` doesn't work with nested outputs so no preparation happens.")
    @slow
```
or
```
    @unittest.skip("Requires an update of the env running those tests")
    @require_torch_multi_gpu
    @require_fairscale
```
or 
```
    @require_tf
    @unittest.skip("Image segmentation not implemented in TF")
```